sys-libs/glibc/files/eblits/src_prepare.eblit

```diff
@@ -58,6 +58,8 @@ eblit-glibc-src_prepare() {
 	find . -type f '(' -size 0 -o -name "*.orig" ')' -delete
 	find . -name configure -exec touch {} +
 
+	eprefixify extra/locale/locale-gen
```

Comment:
We don't ship locale-gen, but I presume this is just modifying the file in ${S} and doesn't impact the installed system right?